src/components/augmentedReality/AugmentedRealityView.js

```diff
@@ -0,0 +1,119 @@
+import {
+  Viro3DObject,
+  ViroAmbientLight,
+  ViroARImageMarker,
+  ViroARScene,
+  ViroARTrackingTargets,
+  ViroSound
+} from '@viro-community/react-viro';
+import PropTypes from 'prop-types';
+import React, { useEffect } from 'react';
+
+import { colors, texts } from '../../config';
+
+export const AugmentedRealityView = ({ sceneNavigator }) => {
+  const {
+    isObjectLoading,
+    setIsObjectLoading,
+    isStartAnimationAndSound,
+    setIsStartAnimationAndSound,
+    object
+  } = sceneNavigator.viroAppProps;
+
+  const TARGET = 'targetImage';
+
+  // TODO: these data can be updated according to the data coming from the server when the
+  //       real 3D models arrive
+  const position = [0, 0, 0];
+  const rotation = [0, 0, 0];
+  const scale = [0.002, 0.002, 0.002];
+
+  useEffect(() => {
+    ViroARTrackingTargets.createTargets({
+      [TARGET]: {
+        physicalWidth: 0.2, // real world width in meters
+        source: { uri: object.target },
+        type: 'image'
+      }
+    });
+  }, [object?.target]);
+
+  const ViroContent = (
+    <ViroSoundAnd3DObject
+      {...{
+        isObjectLoading,
+        isStartAnimationAndSound,
+        object,
+        position,
+        rotation,
+        scale,
+        setIsObjectLoading,
+        setIsStartAnimationAndSound
+      }}
+    />
+  );
+
+  return (
+    <ViroARScene dragType="FixedToWorld">
+      <ViroAmbientLight color={colors.surface} />
+
+      {object?.target ? (
+        <ViroARImageMarker
+          onAnchorFound={() => setIsStartAnimationAndSound(true)} // animation and sound file are started after the image is recognised
+          pauseUpdates // prevents the model from continuous jumping
+          target={TARGET}
+        >
+          {ViroContent}
+        </ViroARImageMarker>
+      ) : (
+        ViroContent
```

Comment:
in this case, sound and animation did not start automatically. do we need to change that? for now i guess no, because we will have targets in any case i think.

Reply:
Yes, I don't think it's necessary for now as we have a target, but a note can be written for the future.